pkgs/development/interpreters/clojurescript/lumo/deps.nix

```diff
@@ -1,13 +1,15 @@
 # generated by clj2nix
-let repos = [
+{ pkgs, ... }:
```

Comment:
It's most common to use `{ pkgs ? import <nixpkgs> {} }:` for such autogenerated files, because then they can be used like `import ./foo.nix {}` but also pass an argument explicitly with `import ./foo.nix { pkgs = ...; }`. And we usually don't use `...` because those extra arguments get ignored.

Reply:
I originally said to use just `pkgs:` and `import ./foo.nix pkgs` [here](https://github.com/NixOS/nixpkgs/pull/44076#discussion_r232508976) because we'd know that this fix is just temporary. I'm suggesting above for it making clj2nix the most usable in general :)

Reply:
ok, l hope this looks about right now?